### ISSUE TITLE
Unable to configure project

### ISSUE THREAD

**Issue by siddhant5exceptions** [breadwallet/breadwallet-android#202]
google-services and mobile-font is empty and also unable to clone project from git facing this error
14:10:24.025: [../../../Wallet] git -c credential.helper= -c core.quotepath=false -c log.showSignature=false clone --progress --recurse-submodules git@github.com:breadwallet/breadwallet-android.git -- breadwallet-android
Cloning into 'breadwallet-android'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
![image_2021_08_31T06_36_22_893Z](https://user-images.githubusercontent.com/89641855/131454094-0eee2f93-1358-4092-9e5e-b441e31446d3.png)

**Comment by devashish-gruslabs** [comment]
I am also facing the same issue. Please revert asap. Also please suggest versions that might be version issues at our end.

Thanks in advance.

**Comment by vinod5ex** [comment]
Yes I also facing the same issue so can any one help me out to resolve it.
